### DISCUSSION TITLE
How to install new releases on MacOS?

### DISCUSSION THREAD

**Discussion by jdomag** [flameshot-org/flameshot#3472] [General]
This is the best screenshot tool I've used, but recently it doesn't start on MacOS, when it does it behaves unstable. I noticed that I use version 12.1.0 which was released 2022. This is the last release according to the Releases on github, but I see a lot of discussions and activities in the repository so I bet this project is not depreciated. 
My question is why there are no newer releases since 2022 or maybe how to install the latest updates? Is there any doc describing the process?

**Comment by mmahmoudian** [comment]
> how to install the latest updates?

You can always try our development builds:

https://flameshot.org/docs/installation/development-build/

Basically, all you need to do it to download the mac version, unpack and install.

> why there are no newer releases since 2022

Long story, so I try to be brief. But before I write it, it might sound like a rant, but it is not :)
Anyways, let's get to the answer of your question. There are multiple reasons:
1. Our lead developer got burnt out and left the project (mostly thanks to certain users with certain attitudes)
2. After Russia attack another dev who was living in Ukraine faced obvious issues and had to focus on family and life
3. People are not putting their money where their mouth is, and we have not received much bounty on issues (bugs or feature requests). More info here: https://flameshot.org/donate/
4. Remaining devs are facing personal life issues and are focused more on their life
5. We asked multiple times for nee devs who want to join the team, but no one showed up

The project is not dead, but we are slightly on back burner until we manage our personal lives.

One very important thing is people start reading our documentation and searching among existing issues before filing a new bug report or feature request. Trying not to get frustrated when replying to the same thing over and over across multiple years for the 70th time take lots of energy and self-control.
